Flagsmith.FlagsmithClient/Flagsmith.FlagsmithClient.csproj

```diff
@@ -6,7 +6,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <PackageId>Flagsmith</PackageId>
     <Title>Flagsmith</Title>
-    <Version>4.3.3</Version>
+    <Version>4.4.0</Version>
```

Comment:
Probably we can just mark this as 4.3.4 - I don't think we're adding extra functionality here that warrants a minor release?